omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -2558,6 +2889,11 @@ def draw_text(self, text, x, y, fontsize, rgb, align="center"):
             x = x - (temp_label.size[0] / 2)
         elif align == "right":
             x = x - temp_label.size[0]
+
+        if align not in ["left-vertical", "right-vertical"]:
+            # The text in TIFF is higher compared to PDF. Add offset
+            y = y + scale_to_export_dpi(1)
+
         x = int(round(x))
```

Comment:
added this because I saw an offset between text in pdf and text in tiff. Need to validate that it works for:
* labels
* scalebar
* colorbar

Reply:
PDF and TIFF look the same after all adjustments of vertical label positions.